ts/parts/Tooltip.ts

```diff
@@ -1385,6 +1414,25 @@ H.Tooltip.prototype = {
                     yAxis.pos + Math.max(0, Math.min(point.plotY, yAxis.len))
             });
         });
+
+        /* If we have a seperate tooltip container, then update the necessary
+         * container properties.
+         * Test that tooltip has its own container and renderer before executing
+         * the operation.
+         */
+        const {
+            container,
+            outside,
+            renderer
+        } = tooltip;
+        if (outside && container && renderer) {
+            updateTooltipContainer(
+                chart.container,
+                container,
+                renderer,
+                tooltipLabel
+            );
+        }
```

Comment:
Seems a little bloaty to pass all the arguments to a private function when we can just a) do it inline or b) use a class function.

Reply:
I see your point here and will adapt this if you believe it is for the better, but in my opinion there is the following downsides with this:

a) do it inline `updateTooltipContainer(tooltip);`
- There is a loss in clarity when doing this. It is more difficult to know what is required for the function to execute when it is asking for a complete `Tooltip` object. Having the required variables as parameters ensures they are passed to the function.
- It is easier to unit-test (if possible) when the function is asking for "only" [`HTMLElement`, `HTMLElement`, `Highcharts.SVGRenderer`, `Highcharts.SVGElement`], which is although more parameters, it is less complicated structures then a complete `Highcharts.Tooltip` object.
- The usability for the function is less flexible as it now requires `Tooltip`. This may seem redundant now, but you never know in the future.

b) use a class function `tooltip.updateTooltipContainer();`
In addition to what I believe is downsides in a) this would also in my opinion add the following
- It bloats the Tooltip prototype as `updateTooltip` is only a utility function and not really meant for the public
- Using context adds state to the function, which again is more difficult to test.

Choosing between these two I will prefer a)

Reply:
I hear you, but I think it's more important that we are consistent. Many functions on the Tooltip property are private by nature. The downside is that we're bloating the prototype, while the upside is that we can hack and fix the Tooltip from the outside.

By doing alternative a) I meant not creating a function at all. Since we're only doing the operation once, there's no real need for calling a function. What we will do in the future isn't relevant now. If we decide we want to reuse this block of code, we can always create a function later.